public/app/plugins/datasource/loki/components/LokiContextUi.tsx

```diff
@@ -136,108 +147,129 @@ export function LokiContextUi(props: LokiContextUiProps) {
   const parsedLabels = contextFilters.filter(({ fromParser }) => fromParser);
   const parsedLabelsEnabled = parsedLabels.filter(({ enabled }) => enabled);
 
+  const contextFilterToSelectFilter = useCallback((contextFilter: ContextFilter): SelectableValue<string> => {
+    return {
+      label: `${contextFilter.label}="${contextFilter.value}"`,
+      value: contextFilter.label,
+    };
+  }, []);
+
   return (
-    <div className={styles.multiSelectWrapper}>
-      <div className={styles.textWrapper}>
-        {' '}
-        <Tooltip
-          content={
-            'This feature is experimental and only works on log queries containing no more than 1 parser (logfmt, json).'
-          }
-          placement="top"
-        >
-          <Tag
-            className={css({
-              fontSize: 10,
-              padding: '1px 5px',
-              verticalAlign: 'text-bottom',
-            })}
-            name={'Experimental'}
-            colorIndex={1}
-          />
-        </Tooltip>{' '}
-        Select labels to be included in the context query:
-        <LoadingPlaceholder text="" className={`${styles.loadingPlaceholder} ${loading ? '' : styles.hidden}`} />
-      </div>
-      <div>
-        <MultiSelect
-          className={styles.multiSelect}
-          prefix="Labels"
-          options={realLabels}
-          value={realLabelsEnabled}
-          formatOptionLabel={formatOptionLabel}
-          closeMenuOnSelect={true}
-          maxMenuHeight={200}
-          menuShouldPortal={false}
-          noOptionsMessage="No further labels available"
-          onChange={(keys, actionMeta) => {
-            if (actionMeta.action === 'select-option') {
-              reportInteraction('grafana_explore_logs_loki_log_context_filtered', {
-                logRowUid: row.uid,
-                type: 'label',
-                action: 'select',
-              });
-            }
-            if (actionMeta.action === 'remove-value') {
-              reportInteraction('grafana_explore_logs_loki_log_context_filtered', {
-                logRowUid: row.uid,
-                type: 'label',
-                action: 'remove',
-              });
+    <div className={styles.wrapper}>
+      <LoadingPlaceholder text="" className={`${styles.loadingPlaceholder} ${loading ? '' : styles.hidden}`} />
```

Comment:
I think we can improve the display of the loading indicator. But that's definitely something for another PR.